InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/Idle.java

```diff
@@ -29,7 +29,7 @@ public void initialize() {
 
     public boolean isFinished() {
         if (DRIVE_SUBSYSTEM.getDriveMethod() == DriveMethod.VELOCITY) {
-            return CommandUtils.stateChange(new VelocityDrive(DRIVE_SUBSYSTEM));
+            return true;
```

Comment:
Can we delete the Idle command now?

Reply:
It is referenced in a few places but I could swap those out with a velocity drive since it's the same thing if we want to get rid of it